docs/site/HasMany-relation.md

```diff
@@ -190,13 +191,13 @@ export class CustomerOrdersController {
 ```
 
 In LoopBack 3, the REST APIs for relations were exposed using static methods
-with the name following the pattern `__{methodName}__{relationName}__`, for
-example `Customer.__find__orders`. We recommend to create a new controller for
-each relation in LoopBack 4 for two reasons. First, it keeps controller classes
-smaller. Second, it creates a logical separation of ordinary repositories and
-relational repositories and thus the controllers which use them. Therefore, as
-shown above, don't add order-related methods to `CustomerController`, but
-instead create a new `CustomerOrdersController` class for them.
+with the name following the pattern `__{methodName}__{relationName}__` (for e.g.
```

Comment:
e.g. stand for `for example` (don't ask me why, probably something to do with latin). Right now this reads `for for example`

Reply:
>It is short for the Latin exempli gratia, “for the sake of example.”

👍